src/plugins/intel_cpu/tests/unit/streams_info/streams_info_table_test.cpp

```diff
@@ -90,7 +94,15 @@ StreamsCalculationTestCase _2sockets_104cores_latency_platform_3 = {
      {52, 26, 0, 26, 1, 0},
      {52, 26, 0, 26, 2, 1},
      {52, 26, 0, 26, 3, 1}},
-    {{1, MAIN_CORE_PROC, 208, -1, -1}},
+    {{1, ALL_PROC, 208, -1, -1},
```

Comment:
There is no ECore used in this situation. Is ALL_PROC also used in this case?

Reply:
Now ALL_PROC is used for all mixed streams, even if the core type is same but numa nodes or sockets are mixed. Then the following function can use it to identify mix stream and parse detail threads on each numa node. 

Changing to MAIN_CORE_PROC is doable. May I know if there is any benefit for StreamExecutor?